docs/lib/storage/fragments/android/getting-started/10_preReq.md

```diff
@@ -1,3 +1,3 @@
 * An Android application targeting Android API level 16 (Android 4.1) or above
-    * For a full example of please follow the [project setup walkthrough](~/lib/project-setup/create-application.md)
+    * For a full example of creating Android project, please follow the [project setup walkthrough](~/lib/project-setup/create-application.md)
```

Comment:
revert this change? would bt inconsistent across all "preReq" sections if u add "creating Andriod Project"

Reply:
Sorry, I don't understand here. I mean iOS, Android and JS have their own 10_preReq.md, right?

Reply:
This is in the Android fragment.